### ISSUE TITLE
New 'Drug Type'

### ISSUE THREAD

**Issue by ecibrian** [chembl/IDG#111]
Could you add in Molecule Dictionaries 'Oligopeptide' as new Drug Type?

**Comment by mnowotka** [comment]
Hi Elena,

The Molecule Dictionary field 'drug type' in the IDG interface is later converted to the 'molecule type' in ChEMBL (confusing - it should have the same name in both places). If I create a new type only in the IDG, we will loose the ability to map our IDG data to ChEMBL. So let's see what @agaulton thinks about this and if there will be a new 'molecule type' at ChEMBL im happy to add the same to the IDG.

**Comment by mnowotka** [comment]
Sorry, I need to check if 'Oligopeptide' is already in ChEMBL but not in the IDG yet.

**Comment by mnowotka** [comment]
```
select distinct(molecule_type) from molecule_dictionary 
```

Gives:

```
Protein
Small molecule
Oligonucleotide
Unclassified
Antibody
Unknown
Enzyme
Oligosaccharide
Cell
```

And they all exist in the IDG interface. So the question to @agaulton stands. Should we add a new 'Oligopeptide' molecule type to ChEMBL?

**Comment by ecibrian** [comment]
I followed the description found in ChEMBL, like in this entry, where the drup type in ‘Oligopeptide’:
https://www.ebi.ac.uk/chembl/compound/inspect/CHEMBL2108078

I would need that type to be able to correctly add the compounds in IDG, cause I still get an error when trying to add it as unknown for example.
Am I wrong about this?

Is there any other way around this issue?   

So the question to @agaulton stands.

**Comment by mnowotka** [comment]
In fact, doing:

```
select molecule_type from molecule_dictionary where chembl_id = 'CHEMBL2108078'
```

returns:

```
Protein
```

I think, that the `Oligopeptide` drug_type displayed on the interface is somehow derived from other molecule data.

**Comment by ecibrian** [comment]
ok, let me try using protein

**Comment by ecibrian** [comment]
It works that way. I will talk to Anna to see if that is ok for IDG, as I guess once the data is migrated to ChEMBL it will appear with the ChEMBL drug type.
Thanks.

**Comment by mnowotka** [comment]
Yes, I will leave that open to keep in mind that this is not yet entirely fixed.
